subprojects/docs/src/snippets/kotlinDsl/creating-tasks-project/groovy/build.gradle

```diff
@@ -1,3 +1,3 @@
-tasks.register('greeting') {
+task greeting {
```

Comment:
why this one does not use `tasks.register`?

Reply:
Because the documentation that has it as a sample talks about that way of creating tasks.

Reply:
see here: https://docs.gradle.org/current/userguide/migrating_from_groovy_to_kotlin_dsl.html#creating_tasks